runtime/src/main/java/org/corfudb/runtime/collections/CMap.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.runtime.collections;
+
+import org.corfudb.annotations.Accessor;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+
+/**
+ * Created by box on 1/25/18.
+ */
+public interface CMap<K, V> extends Map<K, V>{
+    @Accessor
+    public @Nonnull
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)